metpy/plots/__init__.py

```diff
@@ -11,9 +13,16 @@
 from .station_plot import *  # noqa: F403
 from .wx_symbols import *  # noqa: F403
 
+logger = logging.getLogger(__name__)
+
+try:
+    from .cartopy_utils import USCOUNTIES
+except ImportError:
+    logger.warning('Cannot import USCOUNTIES without cartopy installed.')
```

Comment:
So anyone using plots will get this warning if CartoPy is not installed. Is that what we want?

Reply:
What's your suggested alternative? I initially had just a `pass` there, but it seemed like we should give some warning that the functionality is not available.

Reply:
Well, if they don't have CartoPy installed, `add_feature` is going to bork, so I'm not sure how important it is for us to warn, vs. maybe putting it as a `debug`. Then again for us without cartopy `from metpy.plots import USCOUNTIES` will fail. So I'm not sure. On Python 3 (3.7?), we'll have hooks we can use to do something smarter on module attribute access.